### ISSUE TITLE
Update Mpdf Library

### ISSUE THREAD

**Issue by gingebaker** [wanze/Pages2Pdf#10]
Hi. First of all thanks for this great module!

I just wanna let you know that today I tried upgrading the MPDF Library in your module from 5.7.4 to 6.1. ( https://github.com/mpdf/mpdf/tree/6.1 )
It shows now issues here. Everything works as expected. I only added the FileCompiler Tag to mpdf.php.

I personally needed the new list features available from version 6.0 ( https://mpdf.github.io/what-else-can-i-do/lists.html )

Also to mention that I think this would solve the php7 deprecated notices ( #9  ), because they removed the old constructors.

regards, tom

**Comment by wanze** [comment]
@gingebaker That's awesome, thanks for your work! Would you mind opening a Pull request against the dev branch of this repository? I'm happy to do a few quick tests and merge into the master then :)

**Comment by gingebaker** [comment]
Hi wanze! Yes I can do that. just one question:
If I take the whole 6.1 MPDF Library it has about 100MB wich is far more than your 5.7.4 version had. 
I´m not sure but I think you also sorted out some fonts right?
In 6.1 there are also many fonts included in ttfonts directory. I think we should also sort some out.

And there is a new qrcode feature wich also needs about 6MB. I think its better to keep that.

However, should I send you the pull request with the full 6.1 library, and you do the rest?

**Comment by wanze** [comment]
@gingebaker Yes I removed some stuff from the former version as well, the library is just too big.  You can open the pull request with the full library and I'll sort things out. Maybe I can tell mPDF in the new version where to look for additional font files - this would make a great module setting :)
Anyway, thanks again, much appreciated!

**Comment by gingebaker** [comment]
@wanze just sent you the pull request. Hope this works. Lot of files ;) I did read a little about embedding fonts in mpdf, but could not find an official way of doing it without adding fonts directoy to the /ttfonts directory and refer it in config_fonts.php (described here https://mpdf.github.io/fonts-languages/fonts-in-mpdf-6-x.html ). 
I found one "dirty" solution on stackoverflow ( http://stackoverflow.com/questions/17586409/adding-font-to-mpdf ) but I don´t know if that´s the way to go?

**Comment by apeisa** [comment]
Just for the information: current master doesn't work on our server (PHP 7.2.15-0ubuntu0.18.04.1), but the dev branch works. It seems that the [mpdf](https://github.com/mpdf/mpdf/tree/master) runs in 8.00 already, so it would be great to stay with more recent version.
